mods/default/functions.lua

```diff
@@ -721,16 +721,27 @@ end
 
 local log_non_player_actions = minetest.settings:get_bool("log_non_player_actions", false)
 
-function default.log_player_action(player, message, pos)
-	local who = player:get_player_name()
+-- `vector.check` is available since minetest >= 5.5.0
```

Comment:
Since the master of MTG always targets the most recent version (in fact even the dev version), you can rely on `vector.check` and don't need to duplicate it here AFAIK. @sfan5

Reply:
Correct.